android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -160,7 +160,7 @@
 		"xpForNewUnits": 15,
 		"hurryCostModifier": 25,
 		"maintenance": 1,
-		"uniques": ["Culture and Gold costs of acquiring new tiles reduced by 25% in this city"],
+		"uniques": ["-[25]% Culture cost of acquiring tiles [in this city]","-[]% Gold cost of acquiring tiles [in this city]"],
```

Comment:
Forgot the amount in the Gold cost :)
Otherwise looks really good! Thanks for bringing this severely unoptimized unique to my attention :)

Reply:
Typo fixed! Sorry about that one.